containerized_ml_models/model_api_3/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM continuumio/anaconda3
+
+COPY model.pkl api.py requirements.txt /
+
+RUN conda env create --name ml-env --file requirements.txt
+
+ENTRYPOINT [ "conda", "run", "-n", "ml-env", "--no-capture-output", "python", "api.py" ]
```

Comment:
I suggest adding an endline at the end of a file 👺

Reply:
Is there a reason for it?

Reply:
They explain it better than I ever could here: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
That's new, thanks!